extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -466,6 +476,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       [
         DOUBLECLICK_SRA_EXP_BRANCHES.SRA,
         DOUBLECLICK_SRA_EXP_BRANCHES.SRA_NO_RECOVER,
+        RANDOM_SUBDOMAIN_BRANCHES.RANDOM_SUBDOMAIN_CONTROL,
```

Comment:
Ah yeah you don't want this here either :).   Instead place within experimentInfoMap above by adding:
'random-subdomains-for-safeframe': {
   ifTrafficEligible: () => true,
   branches: [RANDOM_SUBDOMAIN_BRANCHES. RANDOM_SUBDOMAIN_CONTROL, RANDOM_SUBDOMAIN_BRANCHES.RANDOM_SUBDOMAIN_EXPERIMENT],
}

Reply:
removed :)